server/channels/app/permissions_migrations.go

```diff
@@ -726,7 +726,7 @@ func (a *App) getAddComplianceSubsectionPermissions() (permissionsMap, error) {
 	// Ancillary permissions
 	transformations = append(transformations, permissionTransformation{
 		On:  permissionExists(model.PermissionSysconsoleWriteComplianceDataRetentionPolicy.Id),
-		Add: []string{model.PermissionCreateDataRetentionJob.Id},
+		Add: []string{model.PermissionCreateDataRetentionJob.Id, model.PermissionManageDataRetentionJob.Id},
```

Comment:
I haven't worked on permission migrations yet. Do they work retroactively, i.e. is `getAddComplianceSubsectionPermissions` running on every server start?

Reply:
It's not, I need to figure out how this gets applied to existing users

Reply:
I need to create a new migration instead of adding to the existing one

Reply:
Added here https://github.com/mattermost/mattermost/pull/26855/commits/522a4a9ee6c61629b8be23faa583a7b128a5b6a5